cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -69,7 +69,7 @@
 {% set nav_items = [(
     ('Consumer Tools', '#', [(
         ('Submit a Complaint', '/complaint/', []),
-        ('Ask CFPB', '/ask-cfpb/' if flag_enabled('WAGTAIL_ASK_CFPB', request) else '/askcfpb/', []),
```

Comment:
Missed this when it was added — I didn't know we could use ternaries in Jinja2 templates!